aip/general/0145.md

```diff
@@ -48,6 +48,25 @@ Exclusive end values are preferable for the following reasons:
 - It is easier to reason about abutting ranges: `[0, x), [x, y), [y, z)`, where
   values are chainable from one range to the next.
 
+### Use of `google.type.Interval`
```

Comment:
For the purposes of the TOC, and the anchor, I think this should be simply,

### Intervals

Reply:
I'd rather make it clear that it's specifically about time (which google.type.Interval is). Otherwise folks might think it includes numeric intervals. How about "Timestamp intervals" or even "Timestamp ranges"?

Reply:
"Timestamp intervals" sgtm

Reply:
Done (locally)